src/main.rs

```diff
@@ -71,45 +72,62 @@ fn encode(words: &[String]) -> (Vec<u32>, HashMap<u32, Vec<String>>) {
 fn encode_word(word: &str) -> u32 {
     let mut code: u32 = 0;
     for char in word.chars() {
-        code |= 1 << (char as u32 - 97);
+        assert!(char.is_ascii_alphabetic());
```

Comment:
I agree such an assert should happen, but shouldn't that be placed in the `load` function? It could either simply drop words that contain anything but ASCII letters or raise an error when I implemented the error handling.

Reply:
Yes that seems fine.